Marlin/language_da.h

```diff
@@ -178,59 +186,55 @@
 
 #define MSG_INFO_MENU                       "Om Printer"
 #define MSG_INFO_PRINTER_MENU               "Printer Info"
-#define MSG_INFO_STATS_MENU                 "Printer Stat."
-#define MSG_INFO_BOARD_MENU                 "Board Info"
+#define MSG_INFO_STATS_MENU                 "Printer Stats"
+#define MSG_INFO_BOARD_MENU                 "Kort Info"
 #define MSG_INFO_THERMISTOR_MENU            "Thermistors"
-#define MSG_INFO_EXTRUDERS                  "Extrudere"
+#define MSG_INFO_EXTRUDERS                  "Extruders"
 #define MSG_INFO_BAUDRATE                   "Baud"
 #define MSG_INFO_PROTOCOL                   "Protocol"
-
-#if LCD_WIDTH > 19
-  #define MSG_INFO_PRINT_COUNT              "Antal Print"
-  #define MSG_INFO_COMPLETED_PRINTS         "Færdige"
-  #define MSG_INFO_PRINT_TIME               "Total print tid"
-  #define MSG_INFO_PRINT_LONGEST            "Længste print tid"
-  #define MSG_INFO_PRINT_FILAMENT           "Total extruderet"
-#else
-  #define MSG_INFO_PRINT_COUNT              "Prints"
-  #define MSG_INFO_COMPLETED_PRINTS         "Færdige"
-  #define MSG_INFO_PRINT_TIME               "Total"
-  #define MSG_INFO_PRINT_LONGEST            "Længste"
-  #define MSG_INFO_PRINT_FILAMENT           "Extruderet"
-#endif
-
+#define MSG_INFO_PRINT_COUNT              "Ant. Prints"
+#define MSG_INFO_COMPLETED_PRINTS         "Færdige"
+#define MSG_INFO_PRINT_TIME               "Total print tid"
+#define MSG_INFO_PRINT_LONGEST            "Længste print"
+#define MSG_INFO_PRINT_FILAMENT           "Total Extruderet"
+#define MSG_INFO_PRINT_COUNT              "Prints"
+#define MSG_INFO_COMPLETED_PRINTS         "Færdige"
+#define MSG_INFO_PRINT_TIME               "Total"
+#define MSG_INFO_PRINT_LONGEST            "Længste"
+#define MSG_INFO_PRINT_FILAMENT           "Extruderet"
```

Comment:
I see you figured out I was referring to
```cpp
#if LCD_WIDTH > 19
  . . .
#else
  . . .
#endif
```
But you neglected to restore the proper indentation.